gamearea.py

```diff
@@ -570,10 +569,10 @@ def save_highscore(self):
     def load_highscore(self):
         file_path = os.path.join(get_activity_root(), 'data', 'highscore')
         if os.path.exists(file_path):
-            try:
-                with open(file_path, "r") as fp:
-                    highscore = fp.readlines()
+            with open(file_path, "r") as fp:
+                highscore = fp.readlines()
+            try: 
                 return int(highscore[0])
-            except(ValueError, IndexError):
+            except (ValueError, IndexError):
                 return 0
```

Comment:
I think that since they're two possible exceptions that can be caught here, it'll be nice to tell which has been caught after either has been caught before returning, a logger error would be fine in this case.